asllib/Interpreter.ml

```diff
@@ -1017,22 +1049,22 @@ module Make (B : Backend.S) (C : Config) = struct
     | S_Decl (_ldk, _ldi, _ty_opt, None) -> fatal_from s TypeInferenceNeeded
     (* End *)
     (* Begin EvalSPrint *)
-    | S_Print { args; newline; debug } ->
-        let* vs = List.map (eval_expr_sef env) args |> sync_list in
+    | S_Print { args = e_list; newline; debug } ->
```

Comment:
This is missing documentation.

Reply:
In the corresponding semantic rules:
SemanticsRule.ECond
SemanticsRule.EArray
SemanticsRule.EArbitrary
SemanticsRule.IsValOfType.INT_WELLCONSTRAINED
SemanticsRule.SFor
SemanticsRule.EvalPattern